and-bible/app/src/main/java/net/bible/android/view/activity/page/screen/DocumentWebViewBuilder.java

```diff
@@ -123,21 +129,38 @@ public void removeWebView(ViewGroup parent) {
 	
 	@SuppressLint("RtlHardcoded")
 	public void addWebView(LinearLayout parent) {
-    	boolean isPortrait = CommonUtils.isPortrait();
+
+		String pref = CommonUtils.getSharedPreference(SPLIT_MODE_PREF, SPLIT_MODE_AUTOMATIC);
+
+		boolean splitHorizontally;
+		switch(pref) {
+			case SPLIT_MODE_AUTOMATIC:
+				splitHorizontally = CommonUtils.isPortrait();
+				break;
+			case SPLIT_MODE_VERTICAL:
+				splitHorizontally = false;
+				break;
+			case SPLIT_MODE_HORIZONTAL:
+				splitHorizontally = true;
+				break;
+			default:
+				throw new RuntimeException("Illegal preference");
+		}
+
     	boolean isWebView = isWebViewShowing(parent);
     	parent.setTag(TAG);
 
     	if (!isWebView || 
     			isWindowConfigurationChanged ||
-    			isPortrait!=isLaidOutForPortrait) {
+    			splitHorizontally!=isLaidOutForPortrait) {

```

Comment:
we should probably standardise naming and rename isLaidOutForPortrait to isLaidOutWithHorizontalSplit or something similar

Reply:
Done